resources/views/admin/blocks/feature_block.blade.php

```diff
@@ -0,0 +1,321 @@
+            'value' => 'digital_publications',
+        ],
+        [
+            'label' => 'Exhibitions',
+            'value' => 'exhibitions',
+        ],
+        [
+            'label' => 'Events',
+            'value' => 'events',
+        ],
+        [
+            'label' => 'Interactive Features',
+            'value' => 'experiences',
+        ],
+        [
+            'label' => 'Highlights',
+            'value' => 'highlights',
+        ],
+        [
+            'label' => 'Videos',
+            'value' => 'videos',
+        ],
+    ]
+])
+
+@formField('radios', [
+    'name' => 'image_ratio',
+    'label' => 'Image Ratio',
+    'default' => 'square',
+    'inline' => true,
+    'options' => [
+        [
+            'value' => 'square',
+            'label' => '1:1'
+        ],
+        [
+            'value' => 'landscape',
+            'label' => '16:9'
+        ],
+    ]
+])
+
+@component('twill::partials.form.utils._connected_fields', [
+    'fieldName' => 'feature_type',
+    'fieldValues' => 'articles',
+    'renderForBlocks' => true
+])
+
+    @formField('select', [
+        'name' => 'columns',
+        'label' => '# of columns',
+        'note' => 'Number of columns selected and items loaded must match',
+        'options' => [
+            [
+                'label' => '2',
+                'value' => 2,
+            ],
+            [
+                'label' => '3',
+                'value' => 3,
+            ],
+            [
+                'label' => '4',
+                'value' => 4,
+            ],
+        ]
+    ])
+
+    @formField('browser', [
+        'name' => 'articles',
+        'endpoints' => [
+                [
+                    'label' => 'Article',
+                    'value' => moduleRoute('articles', 'collection.articles_publications', 'browser', ['is_unlisted' => false]),
+                ],
+            ],
+        'max' => 4,
+        'label' => 'Articles',
+    ])
+
+@endcomponent
+
+@component('twill::partials.form.utils._connected_fields', [
+    'fieldName' => 'feature_type',
+    'fieldValues' => 'digital_publications',
+    'renderForBlocks' => true
+])
+
+    @formField('select', [
+        'name' => 'columns',
+        'label' => '# of columns',
+        'note' => 'Number of columns selected and items loaded must match',
+        'options' => [
+            [
+                'label' => '2',
+                'value' => 2,
+            ],
+            [
+                'label' => '3',
+                'value' => 3,
+            ],
+            [
+                'label' => '4',
+                'value' => 4,
+            ],
+        ]
+    ])
+
+    @formField('browser', [
+        'name' => 'digitalPublications',
+        'endpoints' => [
+                [
+                    'label' => 'Digital Publications',
+                    'value' => moduleRoute('digitalPublications', 'collection.articles_publications', 'browser', ['is_unlisted' => false]),
```

Comment:
While content creators are developing draft content, they may want to link unlisted content that is planned to be listed in the future. Double check with Josh about whether this is a good place to implement this restriction.